ruby-v2/index.rb

```diff
@@ -0,0 +1,7 @@
+require 'algolia'
+
+client = Algolia::Search::Client.create(ENV['ALGOLIA_APPLICATION_ID'], ENV['ALGOLIA_API_KEY'])
```

Comment:
Maybe we should mention in the README how/where to set the ENV values needed? Or at least, mention that we require these to exist and it's up to the end-user how to set them